modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerRuleTest.java

```diff
@@ -11,35 +11,34 @@
 
 public class MockServerContainerRuleTest {
 
-    public static final DockerImageName MOCKSERVER_IMAGE = DockerImageName.parse(
-        "mockserver/mockserver:mockserver-5.14.0"
-    );
-
     // creatingProxy {
+    public static final DockerImageName MOCKSERVER_IMAGE = DockerImageName
+        .parse("mockserver/mockserver")
+        .withTag("mockserver-" + MockServerClient.class.getPackage().getImplementationVersion());
+
     @Rule
-    public MockServerContainer mockServer = new MockServerContainer(
-        MOCKSERVER_IMAGE.withTag("mockserver-" + MockServerClient.class.getPackage().getImplementationVersion())
-    );
+    public MockServerContainer mockServer = new MockServerContainer(MOCKSERVER_IMAGE);
 
     // }
 
     @Test
     public void shouldReturnExpectation() throws Exception {
         // spotless:off
         // testSimpleExpectation {
-        new MockServerClient(mockServer.getHost(), mockServer.getServerPort())
-            .when(request()
+        try(MockServerClient mockServerClient = new MockServerClient(mockServer.getHost(), mockServer.getServerPort())) {
+            mockServerClient.when(request()
                 .withPath("/person")
                 .withQueryStringParameter("name", "peter"))
-            .respond(response()
-                .withBody("Peter the person!"));
+                .respond(response()
+                    .withBody("Peter the person!"));
 
-        // ...a GET request to '/person?name=peter' returns "Peter the person!"
-        // }
-        // spotless:on
+            // ...a GET request to '/person?name=peter' returns "Peter the person!"
+            // }
+            // spotless:on
 
-        assertThat(SimpleHttpClient.responseFromMockserver(mockServer, "/person?name=peter"))
-            .as("Expectation returns expected response body")
-            .contains("Peter the person");
+            assertThat(SimpleHttpClient.responseFromMockserver(mockServer, "/person?name=peter"))
+                .as("Expectation returns expected response body")
+                .contains("Peter the person");
+        }
```

Comment:
test body should look like this

```java
// testSimpleExpectation {
try (
    MockServerClient mockServerClient = new MockServerClient(mockServer.getHost(), mockServer.getServerPort())
) {
    mockServerClient
        .when(request().withPath("/person").withQueryStringParameter("name", "peter"))
        .respond(response().withBody("Peter the person!"));

    // ...a GET request to '/person?name=peter' returns "Peter the person!"

    assertThat(SimpleHttpClient.responseFromMockserver(mockServer, "/person?name=peter"))
        .as("Expectation returns expected response body")
        .contains("Peter the person");
}
// }
```

Couldn't make a suggestion.

Currently, it will display like this

<img width="698" alt="Screenshot 2022-11-01 at 17 03 51" src="https://user-images.githubusercontent.com/1810547/199358682-bda3c26e-daec-4502-8c42-61d0f77a69ea.png">